### PR TITLE
pc - fix bug in message list

### PR DESCRIPTION
**Pull request by pconrad**
Sometimes the user list is not defined and this makes the
component crash. This PR adds a guard against that.

STR: Try opening `member/messages/search` and typing in a search string

OBSERVED: pages crashes (you get a blank page)  (Inspecting console reveals error because `users.length` is undefined.)

DESIRED: page doesn't crash

### CODE REVIEW

**Review by scottpchow23 — DISMISSED** [review]

File: javascript/src/main/components/ChannelMessages/MessageList.js [hunk 1]
```diff
@@ -32,12 +32,12 @@ const messageContents = (text, users) => {
 export default ({ messages, users}) => {
```
Comment (btk5h) [inline javascript/src/main/components/ChannelMessages/MessageList.js:32]: Would it be cleaner to set a default value for the `users` parameter instead? It should be a little less error-prone if more columns are added in the future.
```suggestion
export default ({ messages, users = [] }) => {
```

Comment (scottpchow23) [reply javascript/src/main/components/ChannelMessages/MessageList.js:32]: Ohhh I like this ^

File: javascript/src/main/components/ChannelMessages/MessageList.js [hunk 2]
```diff
@@ -29,15 +29,15 @@ const messageContents = (text, users) => {
 }
 
 
-export default ({ messages, users}) => {
+export default ({ messages, users = [] }) => {
     const columns = [{
         isDummyField: true,
-        formatter: (cell, row) => userName(row.user,users),
+        formatter: (cell, row) => userName(row.user,users||[]),
```
Comment (btk5h) [inline javascript/src/main/components/ChannelMessages/MessageList.js]: Should be unnecessary due to above default.
```suggestion
        formatter: (cell, row) => userName(row.user, users),
```

File: javascript/src/main/components/ChannelMessages/MessageList.js [hunk 3]
```diff
@@ -29,15 +29,15 @@ const messageContents = (text, users) => {
...
         dataField: 'name',
         text: 'Username'
     },{
         isDummyField: true,
-        formatter: (cell, row) => messageContents(row.text, users),
+        formatter: (cell, row) => messageContents(row.text, users||[]),
```
Comment (btk5h) [inline javascript/src/main/components/ChannelMessages/MessageList.js]: Should be unnecessary due to above default.
```suggestion
        formatter: (cell, row) => messageContents(row.text, users),
```

File: javascript/src/main/components/ChannelMessages/MessageList.js (hunk 2 above)
Comment (pconrad) [reply javascript/src/main/components/ChannelMessages/MessageList.js]: done

File: javascript/src/main/components/ChannelMessages/MessageList.js (hunk 3 above)
Comment (pconrad) [reply javascript/src/main/components/ChannelMessages/MessageList.js]: done

**Review by btk5h — APPROVED** [review]

**Pull request merged**